upstairs/src/upstairs.rs

```diff
@@ -1549,7 +1567,7 @@ impl Upstairs {
                     };
                     if should_defer {
                         let tx = self.deferred_msgs.push_oneshot();
-                        rayon::spawn(move || {
```

Comment:
This is the read path, but it looks like we are also using the rayon pool for
decryption as well?  It's fine, but I'm calling false advertising on the PR title :)

Reply:
Yes, blatant false advertising!